neuralmonkey/learning_utils.py

```diff
@@ -209,6 +209,13 @@ def training_loop(tf_manager: TensorFlowManager,
                                         tf_manager.best_score_batch),
                                 color='blue')
 
+                            # store also graph parts
+                            all_coders = set.union(
+                                *[rnr.all_coders for rnr in runners])
```

Comment:
souhlasim, ze se musi ulozit vsechny model_parts, ale nevim jak to napsat elegantne nez tohle cele pak udelat jeste jednou pro trenery a propojit to. Navic to mozna neresi rekurzivni podobu (input>encoder>encoder>decoder>decoder>output) nekterych model_parts.